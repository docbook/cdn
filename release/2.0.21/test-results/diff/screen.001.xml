<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <pre deltaxml:deltaV2="A" class="screen language-none line-numbers">
            <code>(let <a name="dl1" id="dl1"/>
               <span data-number="1" class="callout-bug callout-bug-unicode">❶</span>loopvar <a name="dl2" id="dl2"/>
               <span data-number="2" class="callout-bug callout-bug-unicode">❷</span>((count 1))
  <a name="dl3" id="dl3"/>
               <span data-number="3" class="callout-bug callout-bug-unicode">❸</span>(if (&gt; count 10)
    <a name="dl4" id="dl4"/>
               <span data-number="4" class="callout-bug callout-bug-unicode">❹</span>#t
    (<a name="dl5" id="dl5"/>
               <span data-number="5" class="callout-bug callout-bug-unicode">❺</span>loopvar <a name="dl6" id="dl6"/>
               <span data-number="6" class="callout-bug callout-bug-unicode">❻</span>(+ count 1))))</code>
         </pre><div deltaxml:deltaV2="B" class="content">
            <pre class="screen language-none line-numbers">
               <code>(let <a name="dl1" id="dl1"/>
                  <span data-number="1" class="callout-bug callout-bug-unicode">❶</span>loopvar <a name="dl2" id="dl2"/>
                  <span data-number="2" class="callout-bug callout-bug-unicode">❷</span>((count 1))
  <a name="dl3" id="dl3"/>
                  <span data-number="3" class="callout-bug callout-bug-unicode">❸</span>(if (&gt; count 10)
    <a name="dl4" id="dl4"/>
                  <span data-number="4" class="callout-bug callout-bug-unicode">❹</span>#t
    (<a name="dl5" id="dl5"/>
                  <span data-number="5" class="callout-bug callout-bug-unicode">❺</span>loopvar <a name="dl6" id="dl6"/>
                  <span data-number="6" class="callout-bug callout-bug-unicode">❻</span>(+ count 1))))</code>
            </pre>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>