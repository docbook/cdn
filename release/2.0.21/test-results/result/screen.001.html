<html xmlns="http://www.w3.org/1999/xhtml" xmlns:p="http://www.w3.org/ns/xproc" p:style=""><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Results for screen.001.xml</title><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link rel="stylesheet" type="text/css" href="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/themes/start/jquery-ui.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/prism.css" /><link rel="stylesheet" type="text/css" href="../../resources/css/db-prism.css" /><link href="../style/show-results.css" rel="stylesheet" type="text/css" /><meta name="deltaxml" content="true" /></head><body><h1>DocBook XSLT 2.0 Stylesheet output: screen.001.xml</h1><table class="diffs"><tbody><tr><th>Expected (<span class="exdiff">1</span> differences)</th><th>Actual (<span class="adiff">1</span> differences)</th></tr><tr><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff del">&lt;pre <span class="att">class="screen language-none line-numbers"</span>&gt;
            <span class="tag">&lt;code&gt;(let <span class="tag">&lt;a <span class="att">id="dl1"</span> <span class="att">name="dl1"</span>/&gt;</span>
               <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="1"</span>&gt;❶&lt;/span&gt;</span>loopvar <span class="tag">&lt;a <span class="att">id="dl2"</span> <span class="att">name="dl2"</span>/&gt;</span>
               <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="2"</span>&gt;❷&lt;/span&gt;</span>((count 1))
  <span class="tag">&lt;a <span class="att">id="dl3"</span> <span class="att">name="dl3"</span>/&gt;</span>
               <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="3"</span>&gt;❸&lt;/span&gt;</span>(if (&gt; count 10)
    <span class="tag">&lt;a <span class="att">id="dl4"</span> <span class="att">name="dl4"</span>/&gt;</span>
               <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="4"</span>&gt;❹&lt;/span&gt;</span>#t
    (<span class="tag">&lt;a <span class="att">id="dl5"</span> <span class="att">name="dl5"</span>/&gt;</span>
               <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="5"</span>&gt;❺&lt;/span&gt;</span>loopvar <span class="tag">&lt;a <span class="att">id="dl6"</span> <span class="att">name="dl6"</span>/&gt;</span>
               <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="6"</span>&gt;❻&lt;/span&gt;</span>(+ count 1))))&lt;/code&gt;</span>
         &lt;/pre&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td><td><pre class="show-diff"><span class="tag diff">&lt;html&gt;
   <span class="tag nodiff">&lt;head&gt;
      <span class="tag">&lt;link <span class="att">href="../../resources/css/db-prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/default.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;link <span class="att">href="../../resources/css/prism.css"</span> <span class="att">rel="stylesheet"</span> <span class="att">type="text/css"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">charset="utf-8"</span>/&gt;</span>
      <span class="tag">&lt;meta <span class="att">content="DocBook XSL 2.0 Stylesheets Vxxx"</span> <span class="att">name="generator"</span>/&gt;</span>
      <span class="tag">&lt;script <span class="att">src="../../resources/js/dbmodnizr.js"</span> <span class="att">type="text/javascript"</span>/&gt;</span>
      <span class="tag">&lt;title&gt;Article wrapper&lt;/title&gt;</span>
   &lt;/head&gt;</span>
   <span class="tag diff">&lt;body&gt;
      <span class="tag diff">&lt;article <span class="att">class="article"</span> <span class="att">id="R.1"</span>&gt;
         <span class="tag nodiff">&lt;header <span class="att">class="article-titlepage"</span>&gt;
            <span class="tag">&lt;h2&gt;Article wrapper&lt;/h2&gt;</span>
         &lt;/header&gt;</span>
         <span class="tag diff add">&lt;div <span class="att">class="content"</span>&gt;
            <span class="tag">&lt;pre <span class="att">class="screen language-none line-numbers"</span>&gt;
               <span class="tag">&lt;code&gt;(let <span class="tag">&lt;a <span class="att">id="dl1"</span> <span class="att">name="dl1"</span>/&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="1"</span>&gt;❶&lt;/span&gt;</span>loopvar <span class="tag">&lt;a <span class="att">id="dl2"</span> <span class="att">name="dl2"</span>/&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="2"</span>&gt;❷&lt;/span&gt;</span>((count 1))
  <span class="tag">&lt;a <span class="att">id="dl3"</span> <span class="att">name="dl3"</span>/&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="3"</span>&gt;❸&lt;/span&gt;</span>(if (&gt; count 10)
    <span class="tag">&lt;a <span class="att">id="dl4"</span> <span class="att">name="dl4"</span>/&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="4"</span>&gt;❹&lt;/span&gt;</span>#t
    (<span class="tag">&lt;a <span class="att">id="dl5"</span> <span class="att">name="dl5"</span>/&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="5"</span>&gt;❺&lt;/span&gt;</span>loopvar <span class="tag">&lt;a <span class="att">id="dl6"</span> <span class="att">name="dl6"</span>/&gt;</span>
                  <span class="tag">&lt;span <span class="att">class="callout-bug callout-bug-unicode"</span> <span class="att">data-number="6"</span>&gt;❻&lt;/span&gt;</span>(+ count 1))))&lt;/code&gt;</span>
            &lt;/pre&gt;</span>
         &lt;/div&gt;</span>
      &lt;/article&gt;</span>
      <span class="tag nodiff">&lt;script <span class="att">src="../../resources/js/prism.js"</span>/&gt;</span>
   &lt;/body&gt;</span>
&lt;/html&gt;</span></pre></td></tr><tr><td><em>The “current” CSS styling is applied
                  to the this entire document; the results in this
                  column may be anomalous. Also, image links are broken
                  in this column.</em></td><td> </td></tr><tr><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><pre class="screen language-none line-numbers"><code>(let <a name="dl1" id="dl1"></a><span data-number="1" class="callout-bug callout-bug-unicode">❶</span>loopvar <a name="dl2" id="dl2"></a><span data-number="2" class="callout-bug callout-bug-unicode">❷</span>((count 1))
  <a name="dl3" id="dl3"></a><span data-number="3" class="callout-bug callout-bug-unicode">❸</span>(if (&gt; count 10)
    <a name="dl4" id="dl4"></a><span data-number="4" class="callout-bug callout-bug-unicode">❹</span>#t
    (<a name="dl5" id="dl5"></a><span data-number="5" class="callout-bug callout-bug-unicode">❺</span>loopvar <a name="dl6" id="dl6"></a><span data-number="6" class="callout-bug callout-bug-unicode">❻</span>(+ count 1))))</code></pre></article><script src="../../resources/js/prism.js"></script></td><td><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="content"><pre class="screen language-none line-numbers"><code>(let <a name="dl1" id="dl1"></a><span data-number="1" class="callout-bug callout-bug-unicode">❶</span>loopvar <a name="dl2" id="dl2"></a><span data-number="2" class="callout-bug callout-bug-unicode">❷</span>((count 1))
  <a name="dl3" id="dl3"></a><span data-number="3" class="callout-bug callout-bug-unicode">❸</span>(if (&gt; count 10)
    <a name="dl4" id="dl4"></a><span data-number="4" class="callout-bug callout-bug-unicode">❹</span>#t
    (<a name="dl5" id="dl5"></a><span data-number="5" class="callout-bug callout-bug-unicode">❺</span>loopvar <a name="dl6" id="dl6"></a><span data-number="6" class="callout-bug callout-bug-unicode">❻</span>(+ count 1))))</code></pre></div></article><script src="../../resources/js/prism.js"></script></td></tr></tbody></table><h2>XML source</h2><pre data-src="../src/screen.001.xml"></pre></body><script type="text/javascript" src="http://code.jquery.com/jquery-1.6.4.min.js"></script><script type="text/javascript" src="http://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="../../resources/js/annotation.js"></script><script type="text/javascript" src="../../resources/js/prism.js"></script></html>