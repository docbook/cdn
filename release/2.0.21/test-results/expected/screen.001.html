<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><pre class="screen language-none line-numbers"><code>(let <a name="dl1" id="dl1"></a><span data-number="1" class="callout-bug callout-bug-unicode">❶</span>loopvar <a name="dl2" id="dl2"></a><span data-number="2" class="callout-bug callout-bug-unicode">❷</span>((count 1))
  <a name="dl3" id="dl3"></a><span data-number="3" class="callout-bug callout-bug-unicode">❸</span>(if (&gt; count 10)
    <a name="dl4" id="dl4"></a><span data-number="4" class="callout-bug callout-bug-unicode">❹</span>#t
    (<a name="dl5" id="dl5"></a><span data-number="5" class="callout-bug callout-bug-unicode">❺</span>loopvar <a name="dl6" id="dl6"></a><span data-number="6" class="callout-bug callout-bug-unicode">❻</span>(+ count 1))))</code></pre></article><script src="../../resources/js/prism.js"></script></body></html>